src/main/java/teammates/common/util/Const.java

```diff
@@ -1386,6 +1386,10 @@ private Const() {
                 "Please enter the maximum number of recipients each respondents should give feedback to.";
         public static final String FEEDBACK_QUESTION_TEXTINVALID =
                 "Please enter a valid question. The question text cannot be empty.";
+        public static final String FEEDBACK_QUESTION_MIN_RANK_OPTIONS_EMPTY =
```

Comment:
Sorry I'm confused. This constant is for error message, I don't think it's available anywhere else?

Reply:
Didn't you introduce this in `FeedbackRankQuestionDetails`?

Reply:
My bad, missed that.

Reply:
To make it more consistent, should I remove that one in `FeedbackRankQuestionDetails` and keep the one in `Const` instead?

Reply:
Well, although it makes it more consistent but at the same time `Const` is getting so big with so many irrelevant constants at a single point in time.

The two constants you introduced are really not relevant anywhere else so I thought you encapsulated it nicely in `FeedbackRankQuestionDetails`. In the short term, it might be a little unclear where things are though.

So I'm ok with either case, both seem to have problems and it's not very clear which is better unless `Const` is to be extracted into smaller subparts.

Reply:
I think if we consolidate in `Const`, when we do splitting of `Const` one day, it is more convenient as we know where all of the constants are splitted based on the future design. If we spread some constants everywhere right now, it might cause more trouble in the future as a result?

Reply:
A lot of constants are already all over the place, unfortunately.